.github/actions/get-repo-name/action.yml

```diff
@@ -0,0 +1,13 @@
+name: "crucible-ci get repo name"
+description: "Returns the repository name for the running workflow"
+outputs:
+  repo-name:
+    description: "The repository name for the running workflow"
+    value: ${{ steps.get-repo-name.outputs.repo-name }}
+runs:
+  using: "composite"
+  steps:
+    - name: "get repo name"
+      id: get-repo-name
+      shell: bash
+      run: ${{ github.action_path }}/get-repo-name.sh ${{ github.repository }}
```

Comment:
wondering why you can't simply use github.repository

Reply:
The `github.repository` context variable is of the form `<project>/<repo>` or `<owner>/<repo>` and several of the places that we are wanting to use this information are either a URL or a file path so the presence of the `/` in the string is destructive.  Sometimes we may want to use the repo name in a workflow itself so it is difficult (or at least more difficult than it should be) to do the string parsing natively.  Writing a simple action just makes an easy way to access this information in the format we want directly in a workflow if needed.